Installation/Schema/klant.partij.schema.json

```diff
@@ -26,45 +76,23 @@
       },
       "required": true
     },
-    "interneNotitie": {
-      "type": "string",
-      "maxLength": 200,
-      "required": true
-    },
-    "klanttaakId": {
-      "type": "array",
-      "properties": {
-        "type": "string",
-        "maxLength": 40
-      }
-    },
-    "nummer": {
-      "type": "integer",
-      "required": true
-    },
-    "organisatienaam": {
-      "type": "string",
-      "maxLength": 200
-    },
-    "persoonsnaam": {
+    "vertegenwoordiger": {
       "type": "object",
-      "$ref": "https://commongateway.nl/klant.persoonsnaam.schema.json"
+      "$ref": "https://commongateway.nl/klant.partij.schema.json",
+      "inversedBy": "vertegenwoordigde"
     },
-    "voorkeurskanaal": {
+    "vertegenwoordigde": {
       "type": "object",
-      "description": "Het kanaal waarlangs de partij bij voorkeur interactie heeft met de gemeente.",
-      "$ref": "https://commongateway.nl/klant.voorkeur.schema.json"
+      "$ref": "https://commongateway.nl/klant.partij.schema.json",
+      "inversedBy": "vertegenwoordiger"
```

Comment:
Dubbel check hier, heb je inversedBy aan beide kanten van de relatie geset (als dat nodig is, meestal is dat zo)?

Reply:
ah, ja die zie ik ja. was ook meer een check vraag of je dat niet vergeten was in het algemeen. zag namelijk meerdere inversedby voorbij komen